app/scripts/controllers/swaps.js

```diff
@@ -166,9 +166,9 @@ export default class SwapsController {
       }
     }
 
-    const topAggIdData = await this._findTopQuoteAggId(newQuotes)
-    let { topAggId } = topAggIdData
-    const { isBest } = topAggIdData
+    let topAggIdData
+    let topAggId
+    let isBest
 
     if (isBest) {
```

Comment:
It looks like this should be moved _after_ the following `if {} else {}` blocks.

You copied it into the first block, but left it here where it's always false, and didn't include it in the `else` block

Reply:
Oh, in fact, it's set on line 204 as well. We'll have to reconcile that somehow as well